2023-01-09T05:00:42.538332304Z I0109 05:00:42.538237       1 readyz.go:155] Listening on 0.0.0.0:9980
2023-01-09T05:00:43.191563818Z I0109 05:00:43.191526       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:00:43.193530698Z 2023/01/09 05:00:43 WARNING: [core] [Channel #1 SubChannel #2] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:43.193530698Z   "Addr": "localhost:2379",
2023-01-09T05:00:43.193530698Z   "ServerName": "localhost",
2023-01-09T05:00:43.193530698Z   "Attributes": null,
2023-01-09T05:00:43.193530698Z   "BalancerAttributes": null,
2023-01-09T05:00:43.193530698Z   "Type": 0,
2023-01-09T05:00:43.193530698Z   "Metadata": null
2023-01-09T05:00:43.193530698Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:44.194446690Z 2023/01/09 05:00:44 WARNING: [core] [Channel #1 SubChannel #2] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:44.194446690Z   "Addr": "localhost:2379",
2023-01-09T05:00:44.194446690Z   "ServerName": "localhost",
2023-01-09T05:00:44.194446690Z   "Attributes": null,
2023-01-09T05:00:44.194446690Z   "BalancerAttributes": null,
2023-01-09T05:00:44.194446690Z   "Type": 0,
2023-01-09T05:00:44.194446690Z   "Metadata": null
2023-01-09T05:00:44.194446690Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:45.192763896Z W0109 05:00:45.192524       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T05:00:47.192954533Z I0109 05:00:47.192907       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:00:47.194392937Z 2023/01/09 05:00:47 WARNING: [core] [Channel #3 SubChannel #4] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:47.194392937Z   "Addr": "localhost:2379",
2023-01-09T05:00:47.194392937Z   "ServerName": "localhost",
2023-01-09T05:00:47.194392937Z   "Attributes": null,
2023-01-09T05:00:47.194392937Z   "BalancerAttributes": null,
2023-01-09T05:00:47.194392937Z   "Type": 0,
2023-01-09T05:00:47.194392937Z   "Metadata": null
2023-01-09T05:00:47.194392937Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:48.191920163Z I0109 05:00:48.191883       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:00:48.192837072Z 2023/01/09 05:00:48 WARNING: [core] [Channel #5 SubChannel #6] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:48.192837072Z   "Addr": "localhost:2379",
2023-01-09T05:00:48.192837072Z   "ServerName": "localhost",
2023-01-09T05:00:48.192837072Z   "Attributes": null,
2023-01-09T05:00:48.192837072Z   "BalancerAttributes": null,
2023-01-09T05:00:48.192837072Z   "Type": 0,
2023-01-09T05:00:48.192837072Z   "Metadata": null
2023-01-09T05:00:48.192837072Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:48.195344435Z 2023/01/09 05:00:48 WARNING: [core] [Channel #3 SubChannel #4] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:48.195344435Z   "Addr": "localhost:2379",
2023-01-09T05:00:48.195344435Z   "ServerName": "localhost",
2023-01-09T05:00:48.195344435Z   "Attributes": null,
2023-01-09T05:00:48.195344435Z   "BalancerAttributes": null,
2023-01-09T05:00:48.195344435Z   "Type": 0,
2023-01-09T05:00:48.195344435Z   "Metadata": null
2023-01-09T05:00:48.195344435Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:49.193449286Z W0109 05:00:49.193406       1 etcdcli_pool.go:73] could not create a new cached client after 1 tries, trying again. Err: context deadline exceeded
2023-01-09T05:00:49.193879567Z 2023/01/09 05:00:49 WARNING: [core] [Channel #5 SubChannel #6] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:49.193879567Z   "Addr": "localhost:2379",
2023-01-09T05:00:49.193879567Z   "ServerName": "localhost",
2023-01-09T05:00:49.193879567Z   "Attributes": null,
2023-01-09T05:00:49.193879567Z   "BalancerAttributes": null,
2023-01-09T05:00:49.193879567Z   "Type": 0,
2023-01-09T05:00:49.193879567Z   "Metadata": null
2023-01-09T05:00:49.193879567Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:50.193033527Z W0109 05:00:50.192987       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T05:00:52.193381015Z I0109 05:00:52.193336       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:00:52.194368935Z 2023/01/09 05:00:52 WARNING: [core] [Channel #7 SubChannel #8] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:52.194368935Z   "Addr": "localhost:2379",
2023-01-09T05:00:52.194368935Z   "ServerName": "localhost",
2023-01-09T05:00:52.194368935Z   "Attributes": null,
2023-01-09T05:00:52.194368935Z   "BalancerAttributes": null,
2023-01-09T05:00:52.194368935Z   "Type": 0,
2023-01-09T05:00:52.194368935Z   "Metadata": null
2023-01-09T05:00:52.194368935Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:53.192355913Z I0109 05:00:53.192317       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:00:53.193229674Z 2023/01/09 05:00:53 WARNING: [core] [Channel #9 SubChannel #10] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:53.193229674Z   "Addr": "localhost:2379",
2023-01-09T05:00:53.193229674Z   "ServerName": "localhost",
2023-01-09T05:00:53.193229674Z   "Attributes": null,
2023-01-09T05:00:53.193229674Z   "BalancerAttributes": null,
2023-01-09T05:00:53.193229674Z   "Type": 0,
2023-01-09T05:00:53.193229674Z   "Metadata": null
2023-01-09T05:00:53.193229674Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:53.194286891Z I0109 05:00:53.194258       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:00:53.194984852Z 2023/01/09 05:00:53 WARNING: [core] [Channel #7 SubChannel #8] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:53.194984852Z   "Addr": "localhost:2379",
2023-01-09T05:00:53.194984852Z   "ServerName": "localhost",
2023-01-09T05:00:53.194984852Z   "Attributes": null,
2023-01-09T05:00:53.194984852Z   "BalancerAttributes": null,
2023-01-09T05:00:53.194984852Z   "Type": 0,
2023-01-09T05:00:53.194984852Z   "Metadata": null
2023-01-09T05:00:53.194984852Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:53.195015327Z 2023/01/09 05:00:53 WARNING: [core] [Channel #11 SubChannel #12] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:53.195015327Z   "Addr": "localhost:2379",
2023-01-09T05:00:53.195015327Z   "ServerName": "localhost",
2023-01-09T05:00:53.195015327Z   "Attributes": null,
2023-01-09T05:00:53.195015327Z   "BalancerAttributes": null,
2023-01-09T05:00:53.195015327Z   "Type": 0,
2023-01-09T05:00:53.195015327Z   "Metadata": null
2023-01-09T05:00:53.195015327Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:54.194042612Z W0109 05:00:54.193995       1 etcdcli_pool.go:73] could not create a new cached client after 1 tries, trying again. Err: context deadline exceeded
2023-01-09T05:00:54.194446675Z 2023/01/09 05:00:54 WARNING: [core] [Channel #9 SubChannel #10] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:54.194446675Z   "Addr": "localhost:2379",
2023-01-09T05:00:54.194446675Z   "ServerName": "localhost",
2023-01-09T05:00:54.194446675Z   "Attributes": null,
2023-01-09T05:00:54.194446675Z   "BalancerAttributes": null,
2023-01-09T05:00:54.194446675Z   "Type": 0,
2023-01-09T05:00:54.194446675Z   "Metadata": null
2023-01-09T05:00:54.194446675Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:54.195946506Z 2023/01/09 05:00:54 WARNING: [core] [Channel #11 SubChannel #12] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:54.195946506Z   "Addr": "localhost:2379",
2023-01-09T05:00:54.195946506Z   "ServerName": "localhost",
2023-01-09T05:00:54.195946506Z   "Attributes": null,
2023-01-09T05:00:54.195946506Z   "BalancerAttributes": null,
2023-01-09T05:00:54.195946506Z   "Type": 0,
2023-01-09T05:00:54.195946506Z   "Metadata": null
2023-01-09T05:00:54.195946506Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:55.192951841Z W0109 05:00:55.192910       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T05:00:55.195093446Z W0109 05:00:55.195069       1 etcdcli_pool.go:73] could not create a new cached client after 2 tries, trying again. Err: context deadline exceeded
2023-01-09T05:00:57.193929079Z I0109 05:00:57.193880       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:00:57.194945980Z 2023/01/09 05:00:57 WARNING: [core] [Channel #13 SubChannel #14] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:57.194945980Z   "Addr": "localhost:2379",
2023-01-09T05:00:57.194945980Z   "ServerName": "localhost",
2023-01-09T05:00:57.194945980Z   "Attributes": null,
2023-01-09T05:00:57.194945980Z   "BalancerAttributes": null,
2023-01-09T05:00:57.194945980Z   "Type": 0,
2023-01-09T05:00:57.194945980Z   "Metadata": null
2023-01-09T05:00:57.194945980Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:58.192701211Z I0109 05:00:58.192661       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:00:58.194154828Z I0109 05:00:58.194117       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:00:58.194740143Z 2023/01/09 05:00:58 WARNING: [core] [Channel #15 SubChannel #16] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:58.194740143Z   "Addr": "localhost:2379",
2023-01-09T05:00:58.194740143Z   "ServerName": "localhost",
2023-01-09T05:00:58.194740143Z   "Attributes": null,
2023-01-09T05:00:58.194740143Z   "BalancerAttributes": null,
2023-01-09T05:00:58.194740143Z   "Type": 0,
2023-01-09T05:00:58.194740143Z   "Metadata": null
2023-01-09T05:00:58.194740143Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:58.195363513Z 2023/01/09 05:00:58 WARNING: [core] [Channel #17 SubChannel #18] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:58.195363513Z   "Addr": "localhost:2379",
2023-01-09T05:00:58.195363513Z   "ServerName": "localhost",
2023-01-09T05:00:58.195363513Z   "Attributes": null,
2023-01-09T05:00:58.195363513Z   "BalancerAttributes": null,
2023-01-09T05:00:58.195363513Z   "Type": 0,
2023-01-09T05:00:58.195363513Z   "Metadata": null
2023-01-09T05:00:58.195363513Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:58.195388619Z 2023/01/09 05:00:58 WARNING: [core] [Channel #13 SubChannel #14] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:58.195388619Z   "Addr": "localhost:2379",
2023-01-09T05:00:58.195388619Z   "ServerName": "localhost",
2023-01-09T05:00:58.195388619Z   "Attributes": null,
2023-01-09T05:00:58.195388619Z   "BalancerAttributes": null,
2023-01-09T05:00:58.195388619Z   "Type": 0,
2023-01-09T05:00:58.195388619Z   "Metadata": null
2023-01-09T05:00:58.195388619Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:59.194471694Z W0109 05:00:59.194422       1 etcdcli_pool.go:73] could not create a new cached client after 1 tries, trying again. Err: context deadline exceeded
2023-01-09T05:00:59.195926858Z 2023/01/09 05:00:59 WARNING: [core] [Channel #15 SubChannel #16] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:59.195926858Z   "Addr": "localhost:2379",
2023-01-09T05:00:59.195926858Z   "ServerName": "localhost",
2023-01-09T05:00:59.195926858Z   "Attributes": null,
2023-01-09T05:00:59.195926858Z   "BalancerAttributes": null,
2023-01-09T05:00:59.195926858Z   "Type": 0,
2023-01-09T05:00:59.195926858Z   "Metadata": null
2023-01-09T05:00:59.195926858Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:59.195926858Z 2023/01/09 05:00:59 WARNING: [core] [Channel #17 SubChannel #18] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:59.195926858Z   "Addr": "localhost:2379",
2023-01-09T05:00:59.195926858Z   "ServerName": "localhost",
2023-01-09T05:00:59.195926858Z   "Attributes": null,
2023-01-09T05:00:59.195926858Z   "BalancerAttributes": null,
2023-01-09T05:00:59.195926858Z   "Type": 0,
2023-01-09T05:00:59.195926858Z   "Metadata": null
2023-01-09T05:00:59.195926858Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:00:59.221292585Z I0109 05:00:59.221262       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:00:59.222246837Z 2023/01/09 05:00:59 WARNING: [core] [Channel #19 SubChannel #20] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:00:59.222246837Z   "Addr": "localhost:2379",
2023-01-09T05:00:59.222246837Z   "ServerName": "localhost",
2023-01-09T05:00:59.222246837Z   "Attributes": null,
2023-01-09T05:00:59.222246837Z   "BalancerAttributes": null,
2023-01-09T05:00:59.222246837Z   "Type": 0,
2023-01-09T05:00:59.222246837Z   "Metadata": null
2023-01-09T05:00:59.222246837Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:01:00.194013149Z W0109 05:01:00.193961       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T05:01:00.195052280Z W0109 05:01:00.195031       1 etcdcli_pool.go:73] could not create a new cached client after 2 tries, trying again. Err: context deadline exceeded
2023-01-09T05:01:00.223270695Z 2023/01/09 05:01:00 WARNING: [core] [Channel #19 SubChannel #20] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:01:00.223270695Z   "Addr": "localhost:2379",
2023-01-09T05:01:00.223270695Z   "ServerName": "localhost",
2023-01-09T05:01:00.223270695Z   "Attributes": null,
2023-01-09T05:01:00.223270695Z   "BalancerAttributes": null,
2023-01-09T05:01:00.223270695Z   "Type": 0,
2023-01-09T05:01:00.223270695Z   "Metadata": null
2023-01-09T05:01:00.223270695Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:01:01.222508906Z W0109 05:01:01.222464       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T05:01:02.195141423Z I0109 05:01:02.195092       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:01:02.196347862Z 2023/01/09 05:01:02 WARNING: [core] [Channel #21 SubChannel #22] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:01:02.196347862Z   "Addr": "localhost:2379",
2023-01-09T05:01:02.196347862Z   "ServerName": "localhost",
2023-01-09T05:01:02.196347862Z   "Attributes": null,
2023-01-09T05:01:02.196347862Z   "BalancerAttributes": null,
2023-01-09T05:01:02.196347862Z   "Type": 0,
2023-01-09T05:01:02.196347862Z   "Metadata": null
2023-01-09T05:01:02.196347862Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:01:03.192745875Z I0109 05:01:03.192693       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:01:03.194000291Z 2023/01/09 05:01:03 WARNING: [core] [Channel #23 SubChannel #24] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:01:03.194000291Z   "Addr": "localhost:2379",
2023-01-09T05:01:03.194000291Z   "ServerName": "localhost",
2023-01-09T05:01:03.194000291Z   "Attributes": null,
2023-01-09T05:01:03.194000291Z   "BalancerAttributes": null,
2023-01-09T05:01:03.194000291Z   "Type": 0,
2023-01-09T05:01:03.194000291Z   "Metadata": null
2023-01-09T05:01:03.194000291Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:01:03.195044549Z I0109 05:01:03.195026       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:01:03.195744192Z 2023/01/09 05:01:03 WARNING: [core] [Channel #25 SubChannel #26] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:01:03.195744192Z   "Addr": "localhost:2379",
2023-01-09T05:01:03.195744192Z   "ServerName": "localhost",
2023-01-09T05:01:03.195744192Z   "Attributes": null,
2023-01-09T05:01:03.195744192Z   "BalancerAttributes": null,
2023-01-09T05:01:03.195744192Z   "Type": 0,
2023-01-09T05:01:03.195744192Z   "Metadata": null
2023-01-09T05:01:03.195744192Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:01:03.197130604Z 2023/01/09 05:01:03 WARNING: [core] [Channel #21 SubChannel #22] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:01:03.197130604Z   "Addr": "localhost:2379",
2023-01-09T05:01:03.197130604Z   "ServerName": "localhost",
2023-01-09T05:01:03.197130604Z   "Attributes": null,
2023-01-09T05:01:03.197130604Z   "BalancerAttributes": null,
2023-01-09T05:01:03.197130604Z   "Type": 0,
2023-01-09T05:01:03.197130604Z   "Metadata": null
2023-01-09T05:01:03.197130604Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:01:03.223238298Z I0109 05:01:03.223209       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T05:01:03.224034198Z 2023/01/09 05:01:03 WARNING: [core] [Channel #27 SubChannel #28] grpc: addrConn.createTransport failed to connect to {
2023-01-09T05:01:03.224034198Z   "Addr": "localhost:2379",
2023-01-09T05:01:03.224034198Z   "ServerName": "localhost",
2023-01-09T05:01:03.224034198Z   "Attributes": null,
2023-01-09T05:01:03.224034198Z   "BalancerAttributes": null,
2023-01-09T05:01:03.224034198Z   "Type": 0,
2023-01-09T05:01:03.224034198Z   "Metadata": null
2023-01-09T05:01:03.224034198Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T05:01:04.196532312Z W0109 05:01:04.196484       1 etcdcli_pool.go:73] could not create a new cached client after 1 tries, trying again. Err: context deadline exceeded
